plugin/hooks.go

```diff
@@ -149,4 +153,16 @@ type Hooks interface {
 	// Note that this method will be called for files uploaded by plugins, including the plugin that uploaded the post.
 	// FileInfo.Size will be automatically set properly if you modify the file.
 	FileWillBeUploaded(c *Context, info *model.FileInfo, file io.Reader, output io.Writer) (*model.FileInfo, string)
+
+	// PushNotificationWillBeSent is invoked before the push notification has been sent to the push proxy server.
```

Comment:
The documentation is a bit short, especially about the return value. Please document two things:
- Plugins can modify the notification by changing the return value
- Plugins can reject the notification by returning `nil`

Reply:
Done